data/substructure/substructure.py

```diff
@@ -0,0 +1,11 @@
+# Taken from rdkit Docs
+from rdkit.Chem import rdSubstructLibrary
```

Comment:
You may want to add the other necessary imports, such as:
import os
from rdkit import Chem, RDConfig
import numpy as np